lapis2/src/main/kotlin/org/genspectrum/lapis/util/TryToGuessTheType.kt

```diff
@@ -0,0 +1,41 @@
+package org.genspectrum.lapis.util
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.BooleanNode
+import com.fasterxml.jackson.databind.node.DoubleNode
+import com.fasterxml.jackson.databind.node.JsonNodeFactory
+import com.fasterxml.jackson.databind.node.JsonNodeType.ARRAY
+import com.fasterxml.jackson.databind.node.JsonNodeType.BOOLEAN
+import com.fasterxml.jackson.databind.node.JsonNodeType.NULL
+import com.fasterxml.jackson.databind.node.JsonNodeType.NUMBER
+import com.fasterxml.jackson.databind.node.JsonNodeType.STRING
+import com.fasterxml.jackson.databind.node.LongNode
+import com.fasterxml.jackson.databind.node.TextNode
+import org.genspectrum.lapis.controller.SPECIAL_REQUEST_PROPERTY_TYPES
+
+fun tryToGuessTheType(entry: Map.Entry<String, List<String>>): JsonNode {
```

Comment:
Is this tested somewhere?

Reply:
implicitly, yes. There are e.g. tests that check valid / invalid `offset`s

Reply:
If it is not to much of a hassle, I would favour an explicit test, since this is a core part. So we know that we dont mess up the types.

Reply:
Done